buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -2890,7 +2890,8 @@ class BeamModulePlugin implements Plugin<Project> {
           def cmdArgs = project.project(':sdks:python').mapToArgString(beamPythonTestPipelineOptions)
 
           project.exec {
-            environment "EXPANSION_PORT", externalPort
```

Comment:
could this rename affect existing tests relying on EXPANSION_PORT, or it was just a typo and should be fixed by this change?

Reply:
I only updated "createTransformServiceTask" task, so this should not affect other existing tests.